### DISCUSSION TITLE
Install problem ESP32-C3 (revision 4) with espressif.github.io/esptool-js/ No file selected for row 1!

### DISCUSSION THREAD

**Discussion by newtonsteam** [cmcgerty/Sofar2mqtt#86] [General]
Hello everyone !
Anay idea with this problem when i try to install bins with https://espressif.github.io/esptool-js/

![ESP32-C3](https://github.com/user-attachments/assets/ea4dded8-497a-48d2-88c1-be82e4096b4c)
'No file selected for row 1!'

Many Thanks !

**Comment by cliffdude** [comment]
If you read the project description it says  It's designed to run on an
ESP8266 microcontroller with a TTL to RS485 module such as MAX485 or
MAX3485.
So you need an ESP8266 not ESP32 to use the project without modifiations to
the code.

A quinta, 24/10/2024, 08:05, newtonsteam ***@***.***>
escreveu:
